clients/cpp-console/include/DialogManager.h

```diff
@@ -44,12 +44,17 @@ class DialogManager
     DialogManager(shared_ptr<AgentConfiguration> agentConfig, string audioFilePath);
     const DeviceStatus GetDeviceStatus() { return _deviceStatus; };
     const KeywordActivationState GetKeywordActivationState() { return _keywordActivationState; }
+    //Start a listening session that will terminate after the first utterance. Audio player will be stopped if barge in is supported.
     void StartListening();
+    //Stop audio player, re-initialize back end connection, and restart kws if it is supported.
```

Comment:
here you use "kws" and below you use '"keyword recognition". We need to be consistent. I would call it Keyword Spotting to match the KWS in the API name.

Reply:
Updated.